tests/slow/src/test/scala/tests/sbt/SbtBloopLspSuite.scala

```diff
@@ -350,6 +350,62 @@ class SbtBloopLspSuite
     } yield ()
   }
 
+  test("produce-diagnostics-on-error") {
+    cleanWorkspace()
+    workspace.resolve("sbt").createDirectories()
+    for {
+      _ <- initialize(
+        s"""|/project/build.properties
+            |sbt.version=$sbtVersion
+            |/build.sbt
+            |scalaVersion := "${V.scala213}"
+            |val x: String = 42
+            |""".stripMargin,
+        expectError = true,
+      )
+      _ = assertStatus(!_.isInstalled)
+    } yield {
+      assertNoDiff(
```

Comment:
Could you add a test that makes sure that diagnotics were removed after succesful import?